### PR TITLE
Mention git submodule in README

### PR DESCRIPTION
**Pull request by angelikatyborska**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,16 @@ ElixirAnalyzer.analyze_exercise("two-fer", "./test_data/two_fer/imperfect_soluti
...
+
+Some of the tests depend on the `/elixir` git submodule. Run `git submodule update --init --recursive` once after cloning this repo to also get the submodule.
+
+To check at which commit the submodule is, run `git submodule`. To update the submodule to the newest commit, run `git submodule update --remote --merge`.
+
+The submodule is used only by the tests.
+
+### External tests
+
 However, there are also tests tagged as `:external` which are excluded by default. Those tests check if all of the comments used in this repository exist in [`exercism/website-copy`][website-copy-comments]. To run all tests, use the `--include external` flag.
```
Comment (jiegillet) [inline README.md:71]: It feels strange to start a new section with "However"...

Comment (angelikatyborska) [reply README.md:71]: True, it's not necessary

Comment (angelikatyborska) [inline README.md:71]:
```suggestion
There are also tests tagged as `:external` which are excluded by default. Those tests check if all of the comments used in this repository exist in [`exercism/website-copy`][website-copy-comments]. To run all tests, use the `--include external` flag.
```

**Review by jiegillet — APPROVED** [review]
Nice!
Thank you for updating the submodule too.

**Pull request merged**